src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -20,6 +20,8 @@
 import io.anserini.search.SearchArgs;
 import io.anserini.search.query.BagOfWordsQueryGenerator;
 import io.anserini.search.query.PhraseQueryGenerator;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
Note to @HAKSOAT: Remove these logger imports if not needed in the final implementation.